flink-end-to-end-tests/flink-distributed-cache-via-blob-test/src/main/java/org/apache/flink/streaming/tests/DistributedCacheViaBlobTestProgram.java

```diff
@@ -65,7 +67,11 @@ public static void main(String[] args) throws Exception {
                                 Files.size(inputFile),
                                 inputDir.toAbsolutePath().toString(),
                                 containedFile.getFileName().toString()))
-                .writeAsText(params.getRequired("output"), FileSystem.WriteMode.OVERWRITE);
+                .sinkTo(
+                        FileSink.forRowFormat(
+                                        new org.apache.flink.core.fs.Path(outputPath),
```

Comment:
Remove the FQCN here

Reply:
I did that because of `java.nio.file.Path;`

Reply:
But you're not using the import from nio, so replace it with the one from flink

Reply:
Turns out I am using both!